Use `gh issue edit` instead of `actions/github-script` to add labels in "Adding labels to issues"

jcbhmr opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why:

Closes: 

<!-- If there's an existing issue for your change, please link to it above.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

it's more clear to users -- using CLI tools instead of javascript (another language other than whatever the current project is using) with weird globals that are preassigned. it's much easier to quickly grasp whats going on with the gh cli than to try and track down first the docs for the actions/github-script and then the API docs for `octokit.rest.addLabel()` or whatever.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the preview and current production articles. -->

changing from using the https://github.com/actions/github-script action to using the github cli (gh).

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

jcbhmr committed:
Update adding-labels-to-issues.md

jcbhmr commented:
works great!
![image](https://github.com/github/docs/assets/61068799/50fc3bbe-c51c-4f7a-ad4a-9e80da2d0894)

jcbhmr committed:
find replace "GitHub CLI" to "{% data variables.product.prodname_cli %}"

jcbhmr committed:
explicitly use the title from https://cli.github.com/manual/

hubwriter commented on content/actions/managing-issues-and-pull-requests/adding-labels-to-issues.md:
```diff
@@ -44,20 +44,17 @@ In the tutorial, you will first make a workflow file that uses the [`actions/git
-   - Change the value for `labels` to the list of labels that you want to add to the issue. Separate multiple labels with commas. For example, `["help wanted", "good first issue"]`. For more information about labels, see "[AUTOTITLE](/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-labels-to-issues-and-pull-requests)."
+1. Customize the `env` values in your workflow file:
+   - The `GITHUB_TOKEN`, `GH_REPO`, and `NUMBER` values are automatically set using the `github` and `secrets` contexts. You do not need to change these.
+   - Change the value for `LABELS` to the list of labels that you want to add to the issue. Separate multiple labels with commas. For example, `help wanted,good first issue`. For more information about labels, see "[AUTOTITLE](/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-labels-to-issues-and-pull-requests)."
```
```suggestion
   - Change the value for `LABELS` to the list of labels that you want to add to the issue. The label(s) must exist for your repository. Separate multiple labels with commas. For example, `help wanted,good first issue`. For more information about labels, see "[AUTOTITLE](/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-labels-to-issues-and-pull-requests)."
```

hubwriter committed:
Update content/actions/managing-issues-and-pull-requests/adding-labels-to-issues.md

I've added a sentence to state that the label(s) being added must exist for the repo.

hubwriter reviewed (approved):
I really like this change.

Nice work @jcbhmr! 👏 

I also tested this and it works just fine. I changed one thing: I added a sentence stating that the label(s) the workflow adds must exist for this repo.

I'll get this merged/published. Many thanks for helping improve the docs. Much appreciated.

🚀

hubwriter committed:
Merge branch 'main' into patch-1

waheedullah5961 commented:
Thank you for supporting 👍
